config/routes.rb

```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  root "pages#home"
+  get "about", to: "pages#about"
```

Comment:
`root` tells the Rails route that the default address of the website as `pages#home`.
Use `get` to requests the route to give us the `about` page and send the request to `pages#about`

Reply:
so, what is the "default address" here?

Reply:
since the website is not deployed yet, so the default address is our `127.0.0.1:3000` (`127.0.0.1` is the localhost and `3000` is the port number)

Reply:
or also known as `/` 
This refers to just the root address.